<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>534</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>534</width>
    <height>610</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="LoginLabels" name="lblPictureNolt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>115</y>
     <width>352</width>
     <height>104</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>48000</width>
     <height>48000</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../../images.qrc">:/Windows/Login/Images/loginBackground.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>419</y>
     <width>125</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnLogin
{	
 	border : 1px solid #d0d0d0; 
	color : #d0d0d0;
	background-color:transparent;	
 }

QPushButton#btnLogin:hover
{
 	color:#0992b1;
 } 

QPushButton#btnLogin:pressed
{
 	color:#0992b1;
	border: 1px solid #0992b1; 
} </string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="LoginLineEdits" name="txtLogin">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>327</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #797979;
color : #d0d0d0;
background-color:transparent;
padding-left:7px;</string>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="LoginLineEdits" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>373</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #797979;
color : #d0d0d0;
background-color:transparent;
padding-left:7px;
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="LoginLabels" name="lblLogin">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>332</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:7px;
color : #797979;
background-color:transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="LoginLabels" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>378</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:7px;
color : #797979;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIncorrectLogin">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>465</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:rgb(255,35,6);</string>
   </property>
   <property name="text">
    <string>Invalid credentials</string>
   </property>
  </widget>
  <widget class="LoginLabels" name="lblPictureCat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>524</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>524</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524</width>
     <height>600</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../images.qrc">:/Windows/Login/Images/loginFormCat.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSimpleLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>437</y>
     <width>59</width>
     <height>1</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-top: 1px solid #0992b1;
background:transparent;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>10</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Th</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnQuit{ 
background-color:transparent; 
border:none; 
color : #d0d0d0; 

qproperty-icon: url(&quot; &quot;); 
qproperty-iconSize: 14px ; 
background-image: url(&quot;:/Windows/Login/Images/closeIcon.png&quot;); 
background-repeat: no-repeat; 
background-position:top left;

} 

QPushButton#btnQuit:hover{ 
qproperty-icon: url(&quot; &quot;); 
qproperty-iconSize: 14px 14px; 
background-image: url(&quot;:/Windows/Login/Images/closeIcon.png&quot;); 
background-repeat: no-repeat; 
background-position:top right;
} 

QPushButton#btnQuit:pressed{ 
qproperty-icon: url(&quot; &quot;); 
qproperty-iconSize: 14px 14px; 
background-image: url(&quot;:/Windows/Login/Images/closeIcon.png&quot;); 
background-repeat: no-repeat; 
background-position:top right;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="frame" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>lblPictureCat</zorder>
  <zorder>lblPictureNolt</zorder>
  <zorder>btnLogin</zorder>
  <zorder>txtLogin</zorder>
  <zorder>txtPassword</zorder>
  <zorder>lblLogin</zorder>
  <zorder>lblPassword</zorder>
  <zorder>lblIncorrectLogin</zorder>
  <zorder>txtSimpleLine</zorder>
  <zorder>btnQuit</zorder>
  <zorder>frame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginLabels</class>
   <extends>QLabel</extends>
   <header location="global">CommonUiControllers\Login\LoginLabels.h</header>
  </customwidget>
  <customwidget>
   <class>LoginLineEdits</class>
   <extends>QLineEdit</extends>
   <header location="global">CommonUiControllers\Login\LoginLineEdits.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnLogin</tabstop>
  <tabstop>txtLogin</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>btnQuit</tabstop>
 </tabstops>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
